M2/Macaulay2/e/CMakeLists.txt

```diff
@@ -34,6 +35,9 @@ set(HFILES
 
 # these files are include only.
 set(HPPONLYFILES
+  	MemoryBlock.hpp
+	NCAlgebras/Range.hpp
+	VectorArithmetic2.hpp
```

Comment:
Not urgent, but could you keep to 2 spaces for indentation? That's the common size for CMake.

Reply:
Fixed in https://github.com/Macaulay2/M2/pull/1861/commits/57afec66919d43b7380dcf1ec09ccb3b6ae580b4.